### ISSUE TITLE
Unused argument: custom_parsers

### ISSUE THREAD

**Issue by Fred51** [onnx/sklearn-onnx#1119]
Hello there,

I was trying to see how I would write a custom parser to make it so that a model converted from xgboost classifier could only return probabilities, not the standard label and dictionary of probabilities. I eventually realized it didn't matter what I passed to custom_parsers in convert_sklearn, it returned the same output.

Diving into the code I noticed that, unless I'm missing something, this custom_parsers argument is unused in the parse_sklearn_simple_function.

Is there intention to make use of this parameter? Should it be removed or perhaps an error thrown that custom parsers are currently unsupported?

https://github.com/onnx/sklearn-onnx/blob/99939ef5653ba709716787c934871c33f9557c23/skl2onnx/_parse.py#L104

**Comment by xadupre** [comment]
You can read this example to see how to define your own parser: https://onnx.ai/sklearn-onnx/auto_tutorial/plot_mcustom_parser.html.
